examples/README.md

```diff
@@ -20,3 +19,13 @@ One of the SDK examples covers usage of the mockery within tests.
 ```bash
 go test ./mock/cluster_test.go 
 ```
+
+## Examples Reference
+
+### Retry Example
+
+Example provides automatic retries for all HTTP 500, 429 HTTP status errors.
```

Comment:
@wtrocki @gssbzn note that for instance cluster creation can throw 500 but still created the cluster so next call will be DUPLICATE_CLUSTER_NAME 400 that will be a bit confusing for clients.
retries are mostly useful when operations are atomic

Reply:
yes, personally blindly retrying on the Atlas API is not safe and a bad idea